src/UI/Confetti.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import Confetti from "react-confetti"
+
+const ConfettiForWinner: React.FC = () => {
+
+  return (
+    <div style={{position: 'fixed', left: '350px', top: '47px'}}>
```

Comment:
will be better to put these css properties in a variable

Reply:
Done)